src/TheOpenNetwork/Payloads.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Payloads.h"
+
+
+namespace TW::TheOpenNetwork {
+
+const uint64_t jettonTransferOperation = 0xf8a7ea5;
```

Comment:
Is there any chance this value will be changed in the future?

Reply:
No, it is in jetton standard here https://github.com/ton-blockchain/TEPs/blob/master/text/0074-jettons-standard.md
So it changes only when standard changes
<img width="1026" alt="image" src="https://github.com/trustwallet/wallet-core/assets/17502570/437e1476-1f28-4567-9042-78fd14c62752">